libnavigation-core/src/main/java/com/mapbox/navigation/core/replay/route/ReplayRouteMapper.kt

```diff
@@ -73,7 +73,7 @@ class ReplayRouteMapper @JvmOverloads constructor(
         val replayEvents = mutableListOf<ReplayEventBase>()
         routeLeg.steps()?.flatMap { legStep ->
             val geometry = legStep.geometry() ?: return emptyList()
-            PolylineUtils.decode(geometry, 6)
+            geometry.geometryToPoints(6)
```

Comment:
To the point made above, we can expose a `LegStep.geometryToPoints` extension instead of `String.geometryToPoints`. Same here and everywhere else.